src/platform/platform_internal.h

```diff
@@ -1044,7 +1049,9 @@ QUIC_STATUS
 SocketCreateUdp(
     _In_ CXPLAT_DATAPATH* DataPath,
     _In_ const CXPLAT_UDP_CONFIG* Config,
-    _Out_ CXPLAT_SOCKET** NewSocket
+    _Out_ CXPLAT_SOCKET** NewSocket,
```

Comment:
It is generally recommended to keep out-parameters last. The two bools should be before `NewSocket`